debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/transforms/timescaledb/TimescaleDb.java

```diff
@@ -69,7 +69,7 @@ public R apply(R record) {
 
         final var source = ((Struct) record.value()).getStruct(Envelope.FieldName.SOURCE);
         if (source == null) {
-            LOGGER.debug("Incoming record has an empty source info block {}", record);
```

Comment:
We are not supporting TimeScaleDB for now, so this code path should not executed.
But still fixing the logging here in case we start supporting it down the line. And this was explicitly marked by security too.